### ISSUE TITLE
support for cygwin in windows

### ISSUE THREAD

**Issue by xerron** [blindFS/vim-taskwarrior#79]
I can not run in windows xp sp3.
It would be nice to have it also for windows through cygwin.

**Comment by Chimrod** [comment]
Hello,

I'm using taskwarior in cygwin, and this is how you can make it work with vim.

First, be sure to have « task » in your classpath with ```echo $path```.

Then, in your windows home path, create a file named .taskrc with this contents 

```properties
data.location=/cygdrive/${absolute_path_to_your_home_directory}/.task
```

this way, you do not have to change the HOME variable for vim, and that's all.

**Comment by nowakf** [comment]
Hey, I don't think I've understood the above comment - I get no entries when I try to add something to taskwarrior inside vim, and it says 'unwritten changes', so I guess it must not be getting through to the .task folder - but, I can't seem to get it there.

**Comment by nowakf** [comment]
Just realized it doesn't work in cmder either - so this is definitely not a problem with your plugin - or even Taskwarrior. Ech.

**Comment by nowakf** [comment]
Well, I fixed it. Would explain how, but I'm not sure how it happened. I downloaded cygwin and installed Taskwarrior through that. I added 'cygwin' to the PATH file you get by clicking 'properties>advanced options>enviroment variables' in mycomputer. At this point, TW worked in cmd.exe. Then, I added: "set PATH=%PATH%;C:\cygwin64\bin\" to cmder.exe's enviroment variables. Then I deleted the taskrc I'd made. Now it works. Hopefully somebody who is also not good with computer will find this helpful.
